HongAndJerry/HongAndJerry/Source/Service/CompositionBuilder.swift

```diff
@@ -82,14 +82,17 @@ struct CompositionBuilder {
             }
 
             // 오디오 트랙 추가
-            if let audioTrack = try await asset.loadTracks(withMediaType: .audio).first {
-                // AVFoundation이 자동으로 트랙 ID를 할당하도록 합니다.
-                if let compositionTrack = composition.addMutableTrack(withMediaType: .audio, preferredTrackID: kCMPersistentTrackID_Invalid) {
-                    try compositionTrack.insertTimeRange(timeRange, of: audioTrack, at: .zero)
-                    // 할당된 실제 ID를 배열에 추가합니다.
-                    audioTrackIDs.append(compositionTrack.trackID)
+            if(!segment.isMuted) {
+                if let audioTrack = try await asset.loadTracks(withMediaType: .audio).first {
+                    // AVFoundation이 자동으로 트랙 ID를 할당하도록 합니다.
+                    if let compositionTrack = composition.addMutableTrack(withMediaType: .audio, preferredTrackID: kCMPersistentTrackID_Invalid) {
+                        try compositionTrack.insertTimeRange(timeRange, of: audioTrack, at: .zero)
+                        // 할당된 실제 ID를 배열에 추가합니다.
+                        audioTrackIDs.append(compositionTrack.trackID)
+                    }
                 }
             }
```

Comment:
```suggestion
if !segment.isMuted {
    guard let audioTrack = try await asset.loadTracks(withMediaType: .audio).first else { return }
    guard let compositionTrack = composition.addMutableTrack(
        withMediaType: .audio, 
        preferredTrackID: kCMPersistentTrackID_Invalid
    ) else { return }
    
    try compositionTrack.insertTimeRange(timeRange, of: audioTrack, at: .zero)
    audioTrackIDs.append(compositionTrack.trackID)
}
```
P3
if let 중첩 구문보다 guard let을 두번 쓰는 방식이 가독성과 early return 측면에서 좋을 것 같습니다~

Reply:
동의합니다~!